### PR TITLE
Fix deprecation warnings due to moving to webpack v5

### PR DESCRIPTION
**Pull request by gsavchenko**
A number of plugins that were generated in the original project are not compatible with webpack v5 that results in warning during dev sever startup. I upgraded them and changed the webpack config to get rid of them.

HMR for ts and css still works. But I'm not sure if it's the same. Seems to be 'spammy'.

updated:
Screenshot from 2022-06-09 18-45-01

before:
Screenshot from 2022-06-09 18-51-26

### CODE REVIEW

File: apps/kur_kards_web/assets/package.json [hunk 1]
```diff
@@ -22,17 +22,16 @@
...
     "@types/react-dom": "^18.0.4",
     "babel-loader": "^8.0.0",
-    "copy-webpack-plugin": "^5.1.1",
+    "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^3.4.2",
-    "mini-css-extract-plugin": "^0.9.0",
+    "css-minimizer-webpack-plugin": "^4.0.0",
+    "mini-css-extract-plugin": "^2.6.0",
     "node-sass": "^4.13.1",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
```
Comment (gsavchenko) [inline apps/kur_kards_web/assets/package.json:29]: this is replaced by css-minimizer-webpack-plugin

File: apps/kur_kards_web/assets/package.json [hunk 2]
```diff
@@ -22,17 +22,16 @@
...
     "sass-loader": "^8.0.2",
     "source-map-loader": "^3.0.1",
-    "terser-webpack-plugin": "^2.3.2",
```
Comment (gsavchenko) [inline apps/kur_kards_web/assets/package.json:32]: this comes built into webpack 5.0 now

**Comment by vicovaro** [comment]
I don't see anything wrong, other than you upgraded from packages lol

File: apps/kur_kards_web/assets/package.json [hunk 3]
```diff
@@ -22,17 +22,16 @@
     "@types/react": "^18.0.9",
     "@types/react-dom": "^18.0.4",
     "babel-loader": "^8.0.0",
-    "copy-webpack-plugin": "^5.1.1",
+    "copy-webpack-plugin": "^11.0.0",
```
Comment (vicovaro) [inline apps/kur_kards_web/assets/package.json:25]: huge ass jump right there dawg

**Review by vicovaro — APPROVED** [review]

File: apps/kur_kards_web/assets/package.json (hunk 3 above)
Comment (gsavchenko) [reply apps/kur_kards_web/assets/package.json:25]: fingers crossed nothing blows up

**Pull request merged**